lib/Db/ObjectEntity.php

```diff
@@ -60,14 +60,16 @@ public function hydrate(array $object): self
 
 	public function jsonSerialize(): array
 	{
-		return [
-			'id' => $this->id,
-			'uuid' => $this->uuid,
-			'register' => $this->register,
-			'schema' => $this->schema,
-			'object' => $this->object,
-			'updated' => isset($this->updated) ? $this->updated->format('c') : null,
-			'created' => isset($this->created) ? $this->created->format('c') : null
-		];
+//		return [
+//			'id' => $this->id,
+//			'uuid' => $this->uuid,
+//			'register' => $this->register,
+//			'schema' => $this->schema,
+//			'object' => $this->object,
+//			'updated' => isset($this->updated) ? $this->updated->format('c') : null,
+//			'created' => isset($this->created) ? $this->created->format('c') : null
+//		];
```

Comment:
commented code, we might need this code here ?

Reply:
This is something I need to discuss with Ruben morgen, well spotted, but I had my reasons for now